doc/release-notes/9253-productionPlace.md

```diff
@@ -0,0 +1,5 @@
+## Metadata field Production Location now repeatable, facetable, and enabled for Advanced Search
+This enhancement allows depositors to define multiple instances of the metadata field Production Location in the Citation Metadata block, users to filter search results using the filter facets, and using the field in the Advanced Search option.
+
+## Major Use Cases and Infrastructure Enhancements
+* Data contained in a dataset may have been produced at multiple places. Making the field Production Location repeatable will make it possible to reflect this fact in the dataset metadata. Making the field facetable and enabled for Advanced Search will allow us to customize Dataverse collections more appropriately. (Issue #9253, PR #9254)
```

Comment:
You should include upgrade instructions such as:

### Additional Upgrade Steps

Update the Citation metadata block:

- `wget https://github.com/IQSS/dataverse/releases/download/v5.13/citation.tsv`
- `curl http://localhost:8080/api/admin/datasetfield/load -X POST --data-binary @citation.tsv -H "Content-type: text/tab-separated-values"`

Reply:
@sekmiller took care of this in 5b8e81e. Resolving.